docs/components/docs.jsx

```diff
@@ -55,6 +69,8 @@ class Docs extends React.Component {
               overview={require("!!raw!../ecology-getting-started.md")}
               scope={{React, ReactDOM, V, VictoryChart, VictoryLine, VictoryPie}}
               playgroundtheme="elegant" />
+            <h3>Explore the interactive docs!</h3>
```

Comment:
I didn't want to render old-school hyperlinks to the internal docs (b/c full page load), so I pulled the list out of ecology in order to use react-router `Link` elements.